services/cinema_hall.py

```diff
@@ -0,0 +1,19 @@
+from django.db.models import QuerySet
+
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> QuerySet:
+    queryset = CinemaHall.objects.all()
+
+    return queryset
+
+
+def create_cinema_hall(hall_name: str,
+                       hall_rows: int,
+                       hall_seats_in_row: int
+                       ) -> None:
```

Comment:
```suggestion
def create_cinema_hall(hall_name: str,
                       hall_rows: int,
                       hall_seats_in_row: int
) -> None:
```
Почини форматування тут і в інших місцях, будь ласка. Можеш скористатись black, pylint, flake8, щоб було простіше з форматуванням коду.